client/status/container.go

```diff
@@ -25,6 +32,16 @@ func (x ContainerNotFound) Error() string {
 	)
 }
 
+func (x ContainerNotFound) Is(target error) bool {
```

Comment:
It's an exported method of an exported type -> documentation is needed. Referencing errors.Is is appropriate here.

Reply:
Done